tests/e2e/specs/shopper/active-filters.test.ts

```diff
@@ -66,7 +66,7 @@ const insertBlocks = async () => {
 	await insertBlock( block.name );
 };
 
-const configurateFilterProductsByAttributeBlock = async (
+const configurateFilterByAttributeBlock = async (
```

Comment:
Actually, after a moment I realised that all the names related to Filter by Attribute are named `AttributeFilter`, not `FilterByAttribute`. And that's for all the filters. Having said that, I think it's worth to name this function `configurateAttributeFilterBlock`. That would keep the naming consistent and improves the searching ability, what do you think?

```suggestion
const configureAttributeFilterBlock = async (
```